.github/workflows/updatecli.yaml

```diff
@@ -0,0 +1,28 @@
+name: Updatecli
+on:
+  workflow_dispatch:
+  schedule:
+    # Run every hour
+    - cron: "0 * * * *"
```

Comment:
Hi @olblak ,
I’m wondering why the update is scheduled every hour instead of being triggered on merge.

If the goal is to unify the version, wouldn’t it make more sense to update it only when a bump commit is merged? That way, the version stays in sync with the actual changes and avoids unnecessary updates.

Could you clarify the reasoning behind this schedule?

Thanks!